packages/jsii/lib/validator.ts

```diff
@@ -63,13 +64,26 @@ function _defaultValidations(): ValidationFunction[] {
 
     function _typeNamesMustUsePascalCase(_: Validator, assembly: spec.Assembly, diagnostic: DiagnosticEmitter) {
         for (const type of _allTypes(assembly)) {
+            if (spec.isEnumType(type)) { continue; }
```

Comment:
The enum /type/ should be pascal case, just members should be capital

Reply:
ah i see, totally misread that - so enum values then?
i'll get on it.

Reply:
Enum __members__:

```ts
export enum MyEnum {
  MEMBER_NUMBER_ONE,
  MEMBER_FOO,
  MEMBER_XOO
}
```

Reply:
updated